gadget/install/install.go

```diff
@@ -166,11 +194,17 @@ func Run(model gadget.Model, gadgetRoot, kernelRoot, device string, options Opti
 		// matches what is on the disk, but sometimes there may not be a sector
 		// size specified in the gadget.yaml, but we will always have the sector
 		// size from the physical disk device
-		if err := makeFilesystem(&part, diskLayout.SectorSize); err != nil {
-			return nil, fmt.Errorf("cannot make filesystem for partition %s: %v", part.Role, err)
+		timings.Run(perfTimings, fmt.Sprintf("make-filesystem[%s]", roleOrLabelOrNode(part)), fmt.Sprintf("Create filesystem for %s", roleOrLabelOrNode(part)), func(timings.Measurer) {
```

Comment:
nitpick: I would probably always use part.Node for the "Create filesystem summary for ." summary here